universal-login-react/src/core/utils/isInputAmountUsed.ts

```diff
@@ -0,0 +1,5 @@
+import {TopUpProvider} from '../models/TopUpProvider';
+
+export const isInputAmountUsed = (topUpProvider?: TopUpProvider) => {
```

Comment:
Can you remove it to something like: `providerUsesInputAmount` or `providerAllowsToPrefillAmount` or `amountPrefillableForProvider`